res/online-maps/mapconfig.json

```diff
@@ -1,33 +1,33 @@
 [
     {
-        "name": "OpenStreetMap",
+        "name": "OpenTopoMap",
```

Comment:
The change here is that I moved the "OpenTopoMap" to the top of the map configuration file to use it as default by default even if one enables OpenStreetMap - recall that the first enabled map in this list is now used as the default map app.